### DISCUSSION TITLE
Autenticación sin e-mail

### DISCUSSION THREAD

**Discussion by edwardLz** [supabase/supabase#949] [Questions] [answered]
Hola, estoy recién conociendo supabase, me párese una muy buena herramienta, que me gustaría saber si existe alguna manera de que puede registrar usuarios de autenticación sin necesidad de que tengan email, esto se debe a que en mi aplicación, los usuarios que quieran entrar es muy probable que no cuenten con un email

**Comment by J0** [comment, marked as answer]
Hola,

Gracias por la pregunta. Podría usar authentication con mobil/SMS para registrar una cuenta.

El documentation [está aquí](https://supabase.com/docs/reference/javascript/auth-signup#sign-up-with-phone)

Sorry, my Spanish is terrible so I'll carry on in English. We don't directly support anonymous sign in without email at the moment but there are existing workarounds -- feel free to [read more in this thread](https://github.com/supabase/gotrue/issues/68)

Joel
